aries-site/src/pages/components/tabs.mdx

```diff
@@ -39,9 +41,37 @@ the browser location.** This will allow proper active styling and routing to fun
 ### Alignment
 
 To maintain a clean, consistent presentation, all content should be aligned to the left edge of the Tabs component.
+
 - Apply `justify="start"` to left-align the tabs and have their bottom border extend the full-width of the parent container.
 - Content within the Tab should be flush with the left edge of the Tabs component and have a vertical padding of `medium` to provide spacing and room for the content to breathe.
 
+### Responsive Tabs
+
+When horizontal space is limited, only the Tabs that fit will be displayed. The remaining Tabs can be accessed by using the previous and next arrows or with the keyboard.
+
+#### Responsive Behavior
+
+- Previous and next arrows will be present if the horizontal Tabs are fully outside the viewport.
+- Previous and next arrows are disabled if the horizontal Tab has no more Tabs to view towards the left or right.
+- Clicking on the previous and next arrows will show additional Tabs. At `xsmall` and `small` [breakpoints](/templates/content-layouts?q=responsive#breakpoints) the arrows will show 1 additional Tab. At `medium`, `large`, or `xlarge` [breakpoints](/templates/content-layouts?q=responsive#breakpoints), clicking the arrows will show 3 additional Tabs.
+
+<Notification
+  width="large"
+  status="info"
+  message="Resize your window to see how Tabs adapts when there is not enough room to display all of the Tabs."
+/>
+
+<Example
+  docs="https://v2.grommet.io/tabs?theme=hpe#tabs.step"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tabs/TabResponsive.js"
+  github="https://github.com/grommet/hpe-design-system/blob/master/aries-site/src/examples/components/tabs/TabResponsive.js"
```

Comment:
Need code url placed in an array for the github link to work properly. (didn't know that until just now -- quirk in implementation)

Also modified url for the github link.

```suggestion
  code={[
    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/tabs/TabResponsive.js"
  ]}
  github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/components/tabs/TabResponsive.js"
```